extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/logging/VertxLoggerFormatMapper.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkus.vertx.http.runtime.logging;
+
+import java.util.Optional;
+
+import io.vertx.ext.web.handler.LoggerFormat;
+
+public class VertxLoggerFormatMapper {
```

Comment:
I think this class method contents can be moved to the recorder class (for simplification purposes)